python-flask/app/__init__.py

```diff
@@ -0,0 +1,42 @@
+from distutils.command.config import config
+
+from flask import Flask, session, g
+from app.extensions import db, migrate
+from app.main.routes import main_bp
+from app.posts.routes import posts_bp
+from app.auth.routes import admin_bp
+from app.simulator.routes import sim_bp
+from app.utils import format_datetime, gravatar
+from app.models.user import User
+
+def before_request():
+    """Runs before every request to check if a user is logged in."""
+    g.user = None
+    if 'user_id' in session:
+        g.user = User.query.filter_by(user_id=session['user_id']).first()
+
+def create_app():
+    app = Flask(__name__)
+
+    # Configurations
+    app.config.from_pyfile('../config.py')
+
+    # Initialize extensions
+    db.init_app(app)
+    migrate.init_app(app, db)
```

Comment:
what is this? are you migrating stuff? I would delete it. We shouldn't enable any DB modifications when starting the service. We need to ensure that the service is compatible with the db schema we have

Reply:
Agree with Eduardo here. We need to change this for the two Go-applications as well

Reply:
Addressed and deleted migration stuff.